.github/scripts/release-get-version.sh

```diff
@@ -77,7 +77,20 @@ if [[ -z "$RELEASE_BRANCH_NAME" ]]; then
   exit 1
 fi
 
+# REL_CHANNEL is the release channel (e.g. 0.1, 0.1.0-rc1)
+REL_CHANNEL=""
+
+# If the release version is a release candidate (e.g. 0.1.0-rc1), use the full version as the channel
+# Otherwise, use the major and minor version (e.g. 0.1)
+if [[ "$RELEASE_VERSION" == *"-rc"* ]]; then
+  REL_CHANNEL=$RELEASE_VERSION
+else
+  REL_CHANNEL=$(echo "$RELEASE_VERSION" | cut -d '.' -f 1,2)
+fi
```

Comment:
Can this be edge or latest by any chance? I guess my questions is if this can be anything other than 0.1.0-rc1 or 0.1.0.

Reply:
I see down below that this is just for echoing the value, right?

Reply:
this value comes directly from the `versions.yaml`, so it can't be latest. it will always be `0.1.0` or `0.1.0-rc1`. at the end we're echoing the value out to GITHUB_OUTPUT so that we can use it in the rest of the workflow